app/views/mod_warning/current.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Important message</h1>
+
+<p class="is-lead">Hello <%= current_user.username %>, you have an important message from the <%= SiteSetting['SiteName'] %> community moderation team:</p>
+
+<% if @warning.is_suspension %>
+<div class="notice is-danger">
+    <%= raw(sanitize(@warning_message_html, scrubber: scrubber)) %>
+    <p>Your account has been <strong>temporarily suspended</strong> (ends <span title="<%= current_user.community_user.suspension_end.iso8601 %>">in <%= time_ago_in_words(current_user.community_user.suspension_end) %></span>). We look forwards to your return and continued contributions to the site after this period. In the event of continued rule violations after this period, your account may be removed. If you have any questions about this suspension or would like to dispute it, <a href="mailto:info@codidact.org">contact us</a>.</p>

```

Comment:
Instead of threatening removal, let's just say that further violations can lead to longer suspensions.  (Also, "look forward", not "look forwards".)